staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -115,7 +115,18 @@ func PatchResource(r rest.Patcher, scope RequestScope, admit admission.Interface
 			return nil
 		}
 
-		result, err := patchResource(
+		p := patcher{
+			codec:           codec,
+			namer:           scope.Namer,
+			creater:         scope.Creater,
+			defaulter:       scope.Defaulter,
+			unsafeConvertor: scope.Convertor,
```

Comment:
this changed from UnsafeConvertor to Convertor, right? In patch, we're always working from a copy of the object and don't want to re-deepcopy the input (which is the difference between UnsafeConvertor and Convertor)

Reply:
Unintentional, will revert, good catch.